common/reviews/api/rush-lib.api.md

```diff
@@ -670,7 +670,10 @@ export interface IPhasedCommand extends IRushCommand {
 
 // @public
 export interface IPnpmLockfilePolicies {
-    disallowInsecureSha1?: boolean;
+    disallowInsecureSha1?: {
+        enabled: boolean;
+        skipCheckPackages: Record<string, boolean>;
```

Comment:
1. How about "exemptPackageVersions"?
2. The schema could be
```
"exmeptPackageVersions": {
  "package_name": ["version1", "version2"]
}

Reply:
done